.github/workflows/commit.yaml

```diff
@@ -4,8 +4,11 @@
 ---
 name: "build"
 
-env:  # Update this prior to requiring a higher minor version in go.mod
-  GO_VERSION: "1.17"  # Latest patch
+# GOROOT_NAME resolves a GOROOT path corresponding to a major Golang release on the current runner.
+# Ex. Given GOROOT_NAME=GOROOT_1_17_X64, ${!GOROOT_NAME}=/opt/hostedtoolcache/go/1.17.1/x64
+# See https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#environment-variables-2
+env:
+  GOROOT_NAME: GOROOT_1_17_X64
```

Comment:
so now, everything that talks about go, talks about it in terms of GitHub Actions ENV variables, regardless of if it is building a docker image (prior image) or running go normally.

While not obvious, runners default to different go.. so we still have to do this because runners default to 1.15 even if 1.17 (specifically currently 1.17.1) is installed across the board.